tests/acceptance/FormEditorCreateCustomFieldCest.php

```diff
@@ -47,11 +43,301 @@ public function createCustomSelect(\AcceptanceTester $i) {
     $this->checkCustomSelectInForm($i);
   }
 
+  public function createCustomTextInput(\AcceptanceTester $i) {
+    $i->wantTo('Create custom field: text input');
+    $segmentFactory = new Segment();
```

Comment:
You created a function `editTheForm` which contains recurring code. This is nice. But when I'm looking at the start of new methods then there are 6 still repeating lines. Maybe I'm wrong a this lines have a reason why they aren't in some function. What do you think?

Reply:
Well, never seen this in our acceptance suite, but you're right. Let's try with this now, since it is very frequently used within the tests.